source/features/swap-branches-on-compare.tsx

```diff
@@ -27,5 +27,8 @@ void features.add(__filebasename, {
 	include: [
 		pageDetect.isCompare
 	],
+	exclude: [
+		() => location.pathname.includes('..') && !location.pathname.includes('...')
```

Comment:
Maybe? This makes it slightly more exact because we're specifically excluding 2 dots. That's all. This avoids the awkward `exclude: negative includes`

```suggestion
		() => /\.\.+/.exec(location.pathname)?.[0]!.length === 2
```

Reply:
The reason why I did not use regular expressions is because `includes` method represents intention 
 well. If using regular expression represents intention as well, I agree with your opinion!
Thanks for suggestion.

Reply:
before I change, change PR status to draft.

Reply:
If the suggestion works then its fine. Did you test it?

Reply:
@yakov116 @fregante 
I've just tested, but it was not worked

Reply:
I've just tested. it was worked